### PR TITLE
Two new sushis???

### PR DESCRIPTION
**Pull request by toastywaffelz**
ABOUT

Added two new sushis, the neko roll, and the nekobara roll. Modularized I think??? Not tested at all?? I still have no idea how to format a PR??????

Why is this change beneficial? It's sushi!!!

CHANGELOG
🆑
adds: Adds two new sushi items and sprites, meant to be slightly simpler to craft than other sushi.
/🆑

### CODE REVIEW

File: modular_zubbers/code/modules/food_and_drinks/recipes/tablecraft/recipes_seafood.dm [hunk 1]
```diff
@@ -0,0 +1,143 @@
+/datum/crafting_recipe/food/cubancarp
```
Comment (mcmeiler) [inline modular_zubbers/code/modules/food_and_drinks/recipes/tablecraft/recipes_seafood.dm:1]: Why'd you duplicate a bunch of code from other files here?

File: modular_zubbers/code/game/objects/items/food/misc.dm [hunk 2]
```diff
@@ -0,0 +1,504 @@
+
+////////////////////////////////////////////OTHER////////////////////////////////////////////
+/obj/item/food/watermelonslice
```
Comment (mcmeiler) [inline modular_zubbers/code/game/objects/items/food/misc.dm:3]: ditto

**Review by mcmeiler — CHANGES_REQUESTED** [review]
The PR template is automatically written when you create a new PR - you just deleted it and wrote this, instead.

The PR contains a ton of duplicate recipes in the modular files.

Changelog, please, while you're at it.

**Comment by toastywaffelz** [comment]
lmao I was told to just 'copy the files and their paths' for modularization, nobody told me that I could shave off duplicate code, its done now I think??

**Comment by toastywaffelz** [comment]
OKAY I HAD BRAIN DAMAGE FIXING THIS

but I tested it and can confirm it works!!!!!! YAY!!!

**Review by mcmeiler — APPROVED** [review]

**Review by 567Turtle — APPROVED** [review]
Besides the absolute abhorrent formatting I don't really see anything wrong with this, it's fine

**Pull request merged**